code/datums/quirks/negative_quirks.dm

```diff
@@ -450,6 +450,26 @@
 		quirk_holder.toggle_move_intent()
 	quirk_holder.add_mood_event("nyctophobia", /datum/mood_event/nyctophobia)
 
+/datum/quirk/softspoken
+	name = "Soft-Spoken"
+	desc = "You are soft-spoken, and your voice is hard to hear."
+	icon = FA_ICON_COMMENT
+	value = -2
+	mob_trait = TRAIT_SOFTSPOKEN
+	gain_text = span_danger("You feel like you're speaking more quietly.")
+	lose_text = span_notice("You feel like you're speaking louder.")
+	medical_record_text = "Patient is soft-spoken and difficult to hear."
+
+/datum/quirk/clumsy
+	name = "Clumsy"
+	desc = "You're clumsy, a goofball, a silly dude. You big loveable himbo/bimbo you! Hope you weren't planning on using your hands for anything that takes even a LICK of dexterity."
+	icon = FA_ICON_FACE_DIZZY
+	value = -8
+	mob_trait = TRAIT_CLUMSY
+	gain_text = span_danger("You feel your IQ sink like your brain is liquid.")
+	lose_text = span_notice("You feel like your IQ went up to at least average.")
+	medical_record_text = "Patient is so unbelievably fucking stupid. I've seen clowns with better fine motor skills."
```

Comment:
please make this sound a bit more professional, im not a fan of every medical record entry having some quip at the end.

Reply:
These aren't medical issues though, they're just quirk identifiers, I'm not sure what the point is of making them more true to a medical record when a doctor wouldn't actually note down being bilingual or clumsy anyway.

Reply:
because they appear in the medical record, so either fix that, or fix your description

Reply:
I'm confused why this is crossing a line when we've had a trend of this before.

![image](https://github.com/tgstation/tgstation/assets/98856144/4f1d9f0e-e64f-49e1-a020-1869f4536f41)

![image](https://github.com/tgstation/tgstation/assets/98856144/0745404e-e0c6-4169-86a7-c1d70f390338)


![image](https://github.com/tgstation/tgstation/assets/98856144/82b865fb-d76a-4553-9966-a70f89f41c89)


![image](https://github.com/tgstation/tgstation/assets/98856144/a667176a-de86-4f64-b9af-530256616903)


![image](https://github.com/tgstation/tgstation/assets/98856144/48a203a4-c61d-48fe-878e-392aeccdc31d)


![image](https://github.com/tgstation/tgstation/assets/98856144/f5acdd2c-e577-43c3-b96f-a68c7cf94798)

Reply:
> I'm confused why this is crossing a line when we've had a trend of this before.

Any exception made doesnt excuse every future quirk having some silly quip. I dont particularly care for them either but that doesnt mean i have to allow future quirks having these weird quips.

Reply:
> These aren't medical issues though, they're just quirk identifiers, I'm not sure what the point is of making them more true to a medical record when a doctor wouldn't actually note down being bilingual or clumsy anyway.

true but they are presented in the medical record text. We could just remove non-medical ones from having text to begin with.

Reply:
> > These aren't medical issues though, they're just quirk identifiers, I'm not sure what the point is of making them more true to a medical record when a doctor wouldn't actually note down being bilingual or clumsy anyway.
> 
> true but they are presented in the medical record text. We could just remove non-medical ones from having text to begin with. It is not my right to know what every player has spent their points on.

I don't think that's the fix to this, I corrected the quirk listed above to be more in line with what you listed. However, I think medical should know, or at least be hinted to what is wrong with you. 

If someone comes in unable to talk, or soft-spoken, they should know that it's a quirk screwing them up and not some medical issue. I think less heads will turn and less ahelps will be made if the medical staff always at least has a record that something is up.

Reply:
if you are mute or have things pertinent to medical, then they should obviously stay in the med record text.

Things like bilingual which have no bearing on how you should be treated dont really NEED to be listed.

Reply:
I disagree, without medical records there's no way to know what quirks someone has. This can be very useful info for antagonists, especially those going out of their way to do something fun. "this guy is spiritual? maybe i should pretend to be some kind of deity or supernatural phenomenon."

Also, that's probably out of the scope of this PR, isn't it?

Reply:
stop arguing with the maintainers it will get your PR's merged faster if you just listen to them.